doc/Installation/Installation-Ubuntu-1604-Nginx.md

```diff
@@ -52,14 +52,17 @@ Ensure date.timezone is set in php.ini to your preferred time zone.  See http://
     vim /etc/php/7.0/fpm/php.ini
     vim /etc/php/7.0/cli/php.ini
 
+    vi /etc/php/7.0/fpm/pool.d/www.conf
+    Change the user and group from www-data to librenms
+
     phpenmod mcrypt
     systemctl restart php7.0-fpm
 
 #### Configure NGINX
 
     vim /etc/nginx/conf.d/librenms.conf
 
-Add the following config, edit `server_name` as required:
+Add the following config, edit `server_name` as required, or add _ if you want this vhost to match everything:
```

Comment:
There is nothing special about _, anything that is unresolvable will work.  Could make the default in librenms.conf something unlikely to resolve since many people don't notice they need to edit server_name.
Another item might be to make LibreNMS not user server_name internally so it doesn't break links...

Reply:
Then +1 for librenms not using server_names, if you run it on an internal environment you can't use the ip anymore. This can indeed be removed, as it only breaks librenms when this is the only line you have. What is the reason for librenms auto guessing the url on which it should configure? Why don't you let people just use every hostname they can think of?

Reply:
It can run fine without server_names, just more likely for the user to misconfigure.

Reply:
Librenms uses the servername to redirect you to. If you fill in a servername librenms uses this as a url for the webpage.  In my test env i don't have a hostname pointed to the server, so i have to modify the base url.

Reply:
How does this work then?
![image](https://user-images.githubusercontent.com/39462/57670704-0b077500-75d6-11e9-8457-892f4918a0ad.png)

Reply:
So, i reinstalled librenms again.

Clicking on the alerts tab redirects me to librenms.example.com, clicking on the settings redirects me to librenms.example.com.

Reply:
A lot of the issues with this are fixed by #10227

Reply:
@murrant 👍